crates/ruff_python_formatter/tests/snapshots/black_compatibility@cases__preview_long_strings__regression.py.snap

```diff
@@ -944,12 +944,12 @@ s = f'Lorem Ipsum is simply dummy text of the printing and typesetting industry:
  print(
 -    "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam."
 -    f" {[f'{i}' for i in range(10)]}"
--)
++    f"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam. {[f'{i}' for i in range(10)]}"
```

Comment:
The diff here is slightly confusing. We should double check that this change is correct.